File: Eulerjava/Euler2.java
public class Euler2{
    public static void main(String[] args){
	int current = 2;
	int next = 3;
	int holder = 3;
	int sum = 0;
	while(current < 4000000){
	    if (current % 2 == 0){
		sum += current;
		}
	   holder = next;
	   next += current;
	   current = holder;
	}
	    System.out.println(sum);
    }
}

File: Eulerjs/Euler2.js
var current, next, holder, sum;

current = 2;
next = 3;
holder = 3;
sum = 0;

while(current < 4000000){
	if(!(current % 2)){
    	sum += current;
	}
	holder = next;
	next += current;
	current = holder;
}
console.log(sum);


